cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgisSink.java

```diff
@@ -314,7 +314,6 @@ void persistBatch(NGSIBatch batch)
             aggregator.setEntityForNaming(events.get(0).getEntityForNaming(enableGrouping, enableNameMappings, enableEncoding));
```

Comment:
I guess the following entry in CNR

> [cygnus-ngsi][PostgisSQLSink] Add dm-by-entity-database and dm-by-entity-database-schema

should be modiffied to

> [cygnus-ngsi][PostgresqlSink][PostgisSQLSink] Add dm-by-entity-database and dm-by-entity-database-schema

Reply:
Fixed in [d7cf20a](https://github.com/telefonicaid/fiware-cygnus/pull/1876/commits/d7cf20a4544987e1d98ea28c2147c7c0630df9a7)